jpo-ode-plugins/src/main/java/us/dot/its/jpo/ode/plugin/j2735/J2735HeadingSlice.java

```diff
@@ -0,0 +1,20 @@
+package us.dot.its.jpo.ode.plugin.j2735;
+
+public enum J2735HeadingSlice {
+    from000_0to022_5degrees,
+    from022_5to045_0degrees,
+    from045_0to067_5degrees,
+    from067_5to090_0degrees,
+    from090_0to112_5degrees,
+    from112_5to135_0degrees,
+    from135_0to157_5degrees,
+    from157_5to180_0degrees,
+    from180_0to202_5degrees,
+    from202_5to225_0degrees,
+    from225_0to247_5degrees,
+    from247_5to270_0degrees,
+    from270_0to292_5degrees,
+    from292_5to315_0degrees,
+    from315_0to337_5degrees,
+    from337_5to360_0degrees
+}
```

Comment:
HeadingSlice is a BIT STRING type.   The JER standard writes BITSTRINGs as hex numbers, which is not very human readable, and XER writes that as binary string (one and zeroes).    I do think it makes sense diverge from the JER standard to produce human readable JSON for BITSTRINGs.  But since there is no standard for the text produced, other than what we want to define, they might as well be upper case.